handlers/main.yml

```diff
@@ -1,18 +1,34 @@
 ---
-- name: Build etherpad-lite
+- name: Install the dev depedecencies
   ansible.builtin.shell: |
     pnpm i
-    pnpm run build:etherpad
```

Comment:
I guess this will break. You need to install and build Etherpad with the development dependencies. Running the built Etherpad will work in production mode and without development dependencies.